tests/hacker.test.js

```diff
@@ -55,6 +58,17 @@ describe("POST create hacker", function () {
         });
     });
 
+    it("should FAIL to create a new hacker without the proper token", function(done) {
+        chai.request(server.app)
+            .post(`/api/hacker/`)
+            .type("application/json")
+            .send(newHacker1)
+            .end(function (err, res) {
+                res.should.have.status(422);
+                done();
+            });
+    });
+
```

Comment:
Put in a test for what happens when someone attempts to use an expired token

Reply:
This will be handled in a new PR?

Reply:
yeah

Reply:
Ok, Good to merge